IPython/utils/process.py

```diff
@@ -37,9 +37,8 @@ def find_cmd(cmd):
     is a risk you will find the wrong one.  Instead find those using the
     following code and looking for the application itself::
 
-        from IPython.utils.path import get_ipython_module_path
-        from IPython.utils.process import pycmd2argv
-        argv = pycmd2argv(get_ipython_module_path('IPython.terminal.ipapp'))
+        import sys
+        argv = [sys.executable, '-m', 'IPython.terminal']
```

Comment:
This doesn't actually work:

```
$ python -m IPython.terminal
/usr/bin/python: No module named IPython.terminal.__main__; 'IPython.terminal' is a package and cannot be directly executed
```

The module name needs to be either `IPython` or `IPython.terminal.ipapp`.